src/hooks/useDailyStatistics/useDailyStatistics.test.tsx

```diff
@@ -0,0 +1,247 @@
+import React, { FunctionComponent } from "react";
+import { renderHook } from "@testing-library/react-hooks";
+import { useDailyStatistics } from "./useDailyStatistics";
+import { getDailyStatistics } from "../../services/statistics";
+import { ProductContextProvider } from "../../context/products";
+import { defaultIdentifier } from "../../test/helpers/defaults";
+import { CampaignPolicy } from "../../types";
+import { Sentry } from "../../utils/errorTracking";
+import { wait } from "@testing-library/react-native";
+
+jest.mock("../../utils/errorTracking");
+const mockCaptureException = jest.fn();
+(Sentry.captureException as jest.Mock).mockImplementation(mockCaptureException);
+
+jest.mock("../../services/statistics");
+const mockGetDailyStatistics = getDailyStatistics as jest.Mock;
+
+const key = "KEY";
+const endpoint = "https://myendpoint.com";
+const operatorToken = "operator-token";
+
+const wrapper: FunctionComponent = ({ children }) => (
+  <ProductContextProvider products={customProducts}>
+    {children}
+  </ProductContextProvider>
+);
+
+const customProducts: CampaignPolicy[] = [
+  {
+    category: "tt-token",
```

Comment:
Hmmmm, let's not add this entire policy here, abit sensitive haha. We can rename all categories to `specs` and `lost-specs`? Hahaha what do you think?

Reply:
ooops okay! i took it from `usePastTransaction.test.tsx` HAHA ill change that too!